include/simdutf/implementation.h

```diff
@@ -97,6 +122,19 @@ simdutf_warn_unused size_t convert_valid_utf8_to_utf16(const char * input, size_
  */
 simdutf_warn_unused size_t utf16_length_from_utf8(const char * input, size_t length) noexcept;
 
+/**
+ * Compute the number of 4-byte words that this UTF-8 string would require in UTF-32LE format.
+ *
+ * This function does not validate the input.
+ *
+ * This function is not BOM-aware.
+ *
+ * @param input         the UTF-8 string to process
+ * @param length        the length of the string in bytes
+ * @return the number of char32_t words required to encode the UTF-8 string as UTF-32LE
+ */
+simdutf_warn_unused size_t utf32_length_from_utf8(const char * input, size_t length) noexcept;
```

Comment:
Is this needed as part of the API? That's just count_utf8 isn't it?

Reply:
(I am not asking you to remove the function, I am just curious.)

Reply:
Yes, it is exactly the same. I included it for uniformity, but we can remove it of course...

Reply:
If we keep it, then we should explain in the comments that it is equivalent to... 

See what I mean? Otherwise someone might get confused and think that these are different functions.

Reply:
Ahh yes, that would be clearer indeed.